CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+*                 @afflom @rchincha
```

Comment:
@joshgav, This is displaying an error: Unknown owner on line 4: make sure @rchincha exists and has write access to the repository. 

Might you please point us in the right direction to resolve this?

Reply:
I believe this is due to the teams configuration not being updated, yet. Once this is merged, @joshgav can update the github team to reflect permissions and the error will go away.

Reply:
Accepted the invite, doesn't look to be an issue anymore.